torchdynamo/variables/functions.py

```diff
@@ -284,12 +284,20 @@ def reconstruct(self, codegen):
         if self.kwdefaults:
             flags |= 0x02
             codegen(self.kwdefaults)
-        if isinstance(self.annotations, variables.ConstDictVariable):
+        if isinstance(self.annotations, variables.ConstDictVariable) or isinstance(
```

Comment:
There is a test case covering this, but here we don't require ```annotations``` should be ```ConstDictVariable``` so it doesn't break when I test on Python 3.10. Do we need to add ```assert``` before this line? @jansel

Reply:
This seems like a reasonable fix.  I'll leave it up to you.